lisa/sut_orchestrator/qemu/schema.py

```diff
@@ -33,16 +38,13 @@ class QemuPlatformSchema:
     # An optional remote host for the VMs. All test VMs will be spawned on the
     # specified host by connecting remotely to the libvirt instance running on it.
     # If None, the local machine is used as host.
-    host: Optional[LibvirtHost] = LibvirtHost()
+    host: LibvirtHost = LibvirtHost()
```

Comment:
This must remain `Optional` for back-compat.

Reply:
Default value should ensure backwards compat?

Reply:
It is backward compatible.